### DISCUSSION TITLE
How do I reference SKUs in listings that contain multiple variations?

### DISCUSSION THREAD

**Discussion by erichlin1** [etsy/open-api#1092] [Q&A]
How do I reference SKUs in listings that contain multiple variations?

**Comment by seandmccarthy** [comment]
SKUs are either supplied or obtained from the products array in listing inventory:

https://developers.etsy.com/documentation/reference/#operation/getListingInventory
https://developers.etsy.com/documentation/reference/#operation/updateListingInventory
